modules/cluster-logging-updating-logging.adoc

```diff
@@ -163,3 +185,22 @@ $ oc get CronJob curator -n openshift-logging -o yaml |grep 'image:'
 ----
 image: registry.redhat.io/openshift4/ose-logging-curator5@sha256:330c3499e790d0e184414125a4843cd48849c601eb9f19ff82f30794c858b0bc
 ----
+
+.Post install
+
+After Cluster Logging and Elasticsearch Operators are fully updated to 4.5, you must perform the following actions to complete the upgrade.
+
+* Recreate your Kibana index patterns. Because of changes in Kibana, the cluster logging upgrade does not automatically create index patterns. 
+
+** Use the following command to get a list of new indices in your cluster:
++
+----
+$ oc exec <elasticsearch-pod> -- es_util --query='_cat/indices'
+----
+
+** Users must manually create index patterns to see logs for their projects. Because of the new Elasticsearch data model, all logs that were stored in an index prefixed with *project-* in {product-title} 4.4 are now co-located in a single index prefixed with *app-*. Users should create a new index patterns named `app-` and use the `@timestamp` time field.
+
+** Admin users need to create index patterns using the `app-`, *infra-* and *audit-* prefix with the `@timestamp` time field. The *infra-* indices were prviously *.opertions-*. The *audit-* indices are the audit logs.
```

Comment:
... using the `app`, infra and audit ...

Reply:
@jcantrill Made the suggested changes. Thank you!